guides/common/modules/proc_altering-the-privilege-elevation-method.adoc

```diff
@@ -0,0 +1,20 @@
+[id="altering-the-privilege-elevation-method_{context}"]
+= Altering the Privilege Elevation Method
+
+By default, push-based remote execution uses `sudo` to switch from the SSH user to the effective user that executes the script.
+In some situations, you may require to use another method, such as `su` or `dzdo`.
+You can globally configure an alternative method to switch the effective user in your {Project} settings.
```

Comment:
Maybe mention that "sudo" is the default value?

Reply:
I was trying to do that in the first sentence. Isn't that clear enough?

Reply:
Haha, I've missed that :joy: